src/main/webapp/location-form.html

```diff
@@ -3,15 +3,16 @@
 <head>
     <meta charset="UTF-8">
     <title>location-form</title>
+    <script src="/js/location.js"></script>
 </head>
-<body>
+<body onload="buildUI();">
 
 <div class="padded rounded panel centered">
     <h1 id="page-title">Location Form</h1>
 </div>
 
 <div class="padded rounded panel">
-    <form id="location-form" class="hidden" action = "/location" method="POST" >
+    <form id="location-form" class="hidden" action = "/location" method="POST" enctype = "multipart/form-data">
```

Comment:
Remove the `action = "/location"`, since we're going to set the action from JavaScript. (Having it here is harmless, but people will think "why is it here for".)